src/test/java/Homework20.java

```diff
@@ -0,0 +1,43 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedCondition;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework20 extends BaseTest {
+
+    @Test
+    public void deleteAPlayList() throws InterruptedException{
```

Comment:
Please, delete `throws InterruptedException`

**Thread. sleep is bad! It blocks the current thread and renders it unusable for further work.
Using Thread. sleep() frequently in an automation framework is not a good practice.**

Please use the Explicit Wait command, the WebDriver is directed to wait until a certain condition occurs before proceeding with executing the code.